torch/distributions/binomial.py

```diff
@@ -98,8 +103,11 @@ def log_prob(self, value):
                 self.total_count * torch.log1p((self.logits + 2 * max_val).exp()))
 
     def enumerate_support(self):
-        values = self._new((self.total_count,))
-        torch.arange(self.total_count, out=values.data)
+        total_count = max(int(self.total_count.max().item()), 1)
```

Comment:
this is to ensure that we get valid results when `total_count = 0`.

Reply:
I think this should be `support_size = 1 + int(self.total_count.max().item())`. Sorry I didn't catch this on the first review.